ui_framework/components/color_picker/_color_picker.scss

```diff
@@ -0,0 +1,29 @@
+.kuiColorPicker {
+  cursor: pointer;
```

Comment:
What do you think of adding `display: inline-block` here? This way the hit area of the element is restricted to just the color swatch and hex value. With the current `display` defaulting to `block` the entire horizontal area to the right of the color picker is clickable, even though it's just white space. Changing it to `inline-block` would also make it more consistent with form elements like text inputs.